software/earl/authenticator.go

```diff
@@ -32,6 +32,8 @@ const (
 	AuthExpired          = AuthResult(1)
 	AuthOkButOutsideTime = AuthResult(2) // User ok; time-of-day limit.
 	AuthOk               = AuthResult(42)
+	HolidayHiatusBegin   = 1482192000 // 2016-12-20
```

Comment:
These date are in the middle of the day because of the time zone difference to UTC is this intentional ? Also, the range goes till 16th of January;
So in California timezone, this is 2016-12-19 16:00 .. 2017-01-16 16:00